### ISSUE TITLE
Add support for Organization and Hub users

### ISSUE THREAD

**Issue by javierpf** [FundacionParaguaya/stoplight-web#465]
Add support for hubs and organization users in the stoplight-web.
In the toolbar, they should see all the same options displayed in the PSP platform. Redirect to the platform when users click to any of those options that are not implemented in the stoplight-web.

**Comment by gonzarodriguezt** [comment]
Just to confirm, at the time we don't have implemented or working-on user roles in the app right? @javierpf

**Comment by aprojas** [comment]
This is blocked by another issue @gonzarodriguezt
